stock_putaway_product/demo/product_putaway.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        
+        <record id="product_putaway_per_product"
+                model="product.putaway">
+            <field name="name">Putaway Per Product</field>
```

Comment:
This put away method will ensure that a product can only be received in one fixed location. Perhaps it would be better that in demo, this putaway strategy is called "WH - Putaway Per Product", making reference that it is intended to be used for the "WH" warehouse. 

When I tested I had to create a separate putaway strategy "Chic - Putaway Per Product" in order to be able to receive the same product in a specific location, in "Chic" warehouse as well.